messaging/src/main/java/edu/tamu/weaver/messaging/model/MessageActions.java

```diff
@@ -0,0 +1,5 @@
+package edu.tamu.weaver.messaging.model;
+
+public interface MessageActions {
+
```

Comment:
We will need the generalized DAME message actions here. Do not want to use string literals in some apps and an enum in others.

Reply:
https://github.com/TAMULib/Weaver-Webservice-Core/commit/0136047545c9b6ee1ef128301a3aef53fb55b765